parmed/formats/sdf.py

```diff
@@ -27,17 +28,11 @@ def id_format(filename):
         is_fmt : bool
             True if it is a sdf file, False otherwise
         """
-        f = genopen(filename, 'r')
+        line = linecache.getline(filename, lineno=4)
```

Comment:
Yes I think so. According to what I read: https://en.wikipedia.org/wiki/Chemical_table_file